indico/modules/events/api.py

```diff
@@ -214,11 +220,24 @@ def _addOccurrences(self, fossil, obj, startDT, endDT):
         return fossil
 
     def _makeFossil(self, obj, iface):
+        legacy_obj = obj.as_legacy if isinstance(obj, Event) else obj
         return fossilize(obj, iface, tz=self._tz, naiveTZ=self._serverTZ,
                          filters={'access': self._userAccessFilter},
-                         canModify=obj.canModify(self._aw),
+                         canModify=legacy_obj.canModify(self._aw),
                          mapClassType={'AcceptedContribution': 'Contribution'})
 
+    def _get_query_options(self, include_contribs=False):
+        acl_user_strategy = joinedload('acl_entries').joinedload('user')
+        # remote group membership checks will trigger a load on _all_emails
+        # but not all events use this so there's no need to eager-load them
+        # acl_user_strategy.noload('_primary_email')
+        # acl_user_strategy.noload('_affiliation')
+        creator_strategy = joinedload('creator')
+        contributions_strategy = subqueryload('contributions')
+        if include_contribs:
```

Comment:
You can configure your terminal to show a bigger scrollback buffer. I usually set it to ~100k lines